src/system/group.cr

```diff
@@ -114,7 +114,7 @@ struct System::Group
     @gid <=> other.gid
   end
 
-  # Returns a `String` representation of the group. It's group name.
+  # Returns a `String` representation of the group, its group name.
```

Comment:
I'd change `, its group name.` (it's at least confusing) to: _See `#name`._ (ditto below)